boot/boot.js

```diff
@@ -1963,14 +1963,74 @@ $tw.loadTiddlersFromPath = function(filepath,excludeRegExp) {
 	return tiddlers;
 };
 
+$tw.deferredDirSpecs = [];
+
 /*
 Load all the tiddlers defined by a `tiddlywiki.files` specification file
 filepath: pathname of the directory containing the specification file
 */
-$tw.loadTiddlersFromSpecification = function(filepath,excludeRegExp) {
+$tw.loadTiddlersFromSpecification = function(filepath,excludeRegExp,loadDeferred) {
```

Comment:
I would suggest that we use `function(filepath,excludeRegExp,options) {` as a new parameter. So we do not have to change the function-signature again. It's easy to add new options in the future. eg:

```
$tw.loadTiddlersFromSpecification = function(filepath,excludeRegExp,loadDeferred) {
	options = options || {};
```

So you can use `options.loadDeferred` -- Look at line 2259 ff ($tw.loadWikiTiddlers) to see how we document it.